src/apicompat/baselines/ApiCompatBaseline.netstandard2.0.txt

```diff
@@ -1,8 +1,105 @@
+Compat issues with assembly mscorlib:
+TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref struct' in the implementation but is a 'struct' in the contract.
+TypeCannotChangeClassification : Type 'System.TypedReference' is a 'ref struct' in the implementation but is a 'struct' in the contract.
+Compat issues with assembly netstandard:
+TypeCannotChangeClassification : Type 'System.RuntimeArgumentHandle' is a 'ref struct' in the implementation but is a 'struct' in the contract.
+TypeCannotChangeClassification : Type 'System.TypedReference' is a 'ref struct' in the implementation but is a 'struct' in the contract.
+CannotSealType : Type 'System.ComponentModel.BaseNumberConverter' is effectively (has a private constructor) sealed in the implementation but not sealed in the contract.
```

Comment:
It may be worth adding functionality to APICompat so that we can produce a baseline file that doesn't have these duplicate entries (in this case, one each for `netstandard`, `system`, and `System.ComponentModel`). We could separate the concept of baseline files (like this one) with dev-friendly baseline files (without the duplicates). APICompat reads the former, and writes both.